### PR TITLE
Implement occurrence and schedule API(s)

### PR DESCRIPTION
**Pull request by tteerawat**
Create 2 new modules `Occurrence` and `Schedule` and add new functions to `Charge`and `Customer` modules

### CODE REVIEW

File: lib/omise/charge.ex [hunk 1]
```diff
@@ -305,4 +305,26 @@ defmodule Omise.Charge do
...
+    * `limit` - (optional, default: 20, maximum: 100) The maximum amount of records returned.
+    * `from` - (optional, default: 1970-01-01T00:00:00Z, format: ISO 8601) The UTC date and time limiting the beginning of returned records.
+    * `to` - (optional, default: current UTC Datetime, format: ISO 8601) The UTC date and time limiting the end of returned records.
+
+  ## Examples
+
+      Omise.Charge.list_schedules
+
+  """
+  @spec list_schedules(Keyword.t, Keyword.t) :: {:ok, Omise.List.t} | {:error, Omise.Error.t}
```
Comment (zentetsukenz) [inline lib/omise/charge.ex:325]: Might be better to add `| none()` type to the parameter list since caller can omit it na.

**Review by zentetsukenz — COMMENTED** [review]
Only have comment for the type specs. Otherwise, looks good to me 👍

File: lib/omise/charge.ex (hunk 1 above)
Comment (zentetsukenz) [reply lib/omise/charge.ex:325]: 🤔  I think maybe not since `Keyword.t` already has `any()` type.

**Review by zentetsukenz — APPROVED** [review]

File: lib/omise/charge.ex (hunk 1 above)
Comment (tteerawat) [reply lib/omise/charge.ex:325]: 🤔

**Pull request merged**